### ISSUE TITLE
## my train cannot converge？！

### ISSUE THREAD

**Issue by seasonyang** [balancap/SSD-Tensorflow#124]
## my train cannot converge？！
**I have only one classes to deteted which is diffrent from all VOC dataset，my training dataset is 1500 pictures，and I train it from pre-trained vgg_16。
throug training 25000 steps the loss still nearby 4.x，why it doen’t reduce lower？**

**the script for training is：**
```
DATASET_DIR=./tfrecords/voc2007

TRAIN_DIR=./logs/my_chkp

CHECKPOINT_PATH=./checkpoints/vgg_16.ckpt

python3.4 train_ssd_network.py \
    --train_dir=${TRAIN_DIR} \
    --dataset_dir=${DATASET_DIR} \
    --dataset_name=pascalvoc_2007 \
    --dataset_split_name=train \
    --model_name=ssd_300_vgg \
    --checkpoint_path=${CHECKPOINT_PATH} \
    --checkpoint_model_scope=vgg_16 \
    --checkpoint_exclude_scopes=ssd_300_vgg/conv6,ssd_300_vgg/conv7,ssd_300_vgg/block8,ssd_300_vgg/block9,ssd_300_vgg/block10,ssd_300_vgg/block11,ssd_300_vgg/block4_box,ssd_300_vgg/block7_box,ssd_300_vgg/block8_box,ssd_300_vgg/block9_box,ssd_300_vgg/block10_box,ssd_300_vgg/block11_box \
    --trainable_scopes=ssd_300_vgg/conv6,ssd_300_vgg/conv7,ssd_300_vgg/block8,ssd_300_vgg/block9,ssd_300_vgg/block10,ssd_300_vgg/block11,ssd_300_vgg/block4_box,ssd_300_vgg/block7_box,ssd_300_vgg/block8_box,ssd_300_vgg/block9_box,ssd_300_vgg/block10_box,ssd_300_vgg/block11_box \
    --save_summaries_secs=60 \
    --save_interval_secs=600 \
    --weight_decay=0.0005 \
    --optimizer=adam \
    --learning_rate=0.001 \
    --learning_rate_decay_factor=0.94 \
    --batch_size=64
```

**## training steps:**
_INFO:tensorflow:Recording summary at step 26455.
INFO:tensorflow:global step 26460: loss = 4.0582 (1.373 sec/step)
INFO:tensorflow:global step 26470: loss = 4.0544 (1.371 sec/step)
INFO:tensorflow:global step 26480: loss = 4.0573 (1.374 sec/step)
INFO:tensorflow:global step 26490: loss = 4.0581 (1.702 sec/step)
INFO:tensorflow:Recording summary at step 26498.
INFO:tensorflow:global step 26500: loss = 4.0551 (1.374 sec/step)
INFO:tensorflow:global step 26510: loss = 4.0563 (1.376 sec/step)
INFO:tensorflow:global step 26520: loss = 4.0573 (1.375 sec/step)_

**Comment by congjianting** [comment]
@seasonyang Hi, I face the same problem,  do you find the solution?

**Comment by zoumengchao** [comment]
the result is very well     as my loss always  up and down   at   40   while my step over 100000+

**Comment by zoumengchao** [comment]
@congjianting   new a also fine the same problem ,do you find the solution.   my data ,methon. keep all the same with  balancap

**Comment by ivyteng** [comment]
@zoumengchao  I face the same problem, also up and down at 40. Have you solved the problem?
